src/frontend/youtube_clone/src/app/profile-page/profile-page.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Location } from '@angular/common';
+
+
+@Component({
+  selector: 'app-profile-page',
+  templateUrl: './profile-page.component.html',
+  styleUrls: ['./profile-page.component.scss']
+})
+export class ProfilePageComponent {
+  constructor(private route: ActivatedRoute, private location: Location) { }
```

Comment:
Why do you need `private route: ActivatedRoute, private location: Location`?

Reply:
I was trying to make the routing work, when i understood i removed it from some components, but forgot to remove it in some others.
 I fixed it now.